run

```diff
@@ -28,8 +28,9 @@ if ! diff  <(cat .nvmrc | tr -d '\n') <(node -v | tr -d '\n') > /dev/null ; then
     exit 1
 fi
 
-# check yarn exists
-require_cmd yarn "On macOS, install with 'brew install yarn'"
+# check yarn exists with corepack
+require_cmd corepack "Please enable it with 'corepack enable'"
```

Comment:
Will this instruction work? Just thinking that if don't you have the command corepack then running `corepack enable` won't work. I could be missing something though.